README.md

```diff
@@ -1 +1,180 @@
-# BUDGET-APP
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+<details>
+  <summary>📗 Table of Contents</summary>
+  <ul>
+    <li><a href="#-budget-app-">📖 Budget App</a></li>
+    <li><a href="#-built-with-">🛠 Built With</a></li>
+    <li><a href="#-getting-started-">💻 Getting Started</a></li>
+    <li><a href="#-authors-">👥 Authors </a></li>
+    <li><a href="#-future-features-">🔭 Future Features</a></li>
+    <li><a href="#-contributing-">🤝 Contributing</a></li>
+    <li><a href="#️-show-your-support-">⭐️ Show your support </a></li>
+    <li><a href="#-acknowledgments-">🙏 Acknowledgments </a></li>
+    <li><a href="#-license-">📝 License</a></li>
+  </ul>
+</details>
+
+<br>
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Budget App <a name="about-project"></a>
+
+**Budget App** is a Ruby on Rails project designed to help you monitor you spending habits. Go ahead and sign up to start organizing your transactions, monitoring your total spend and keeping track of your budget.
+![image](https://github.com/stino-x/BUDGET-APP/assets/107426247/411fdc51-91ca-4a23-8c8e-21ff46a4af29)
+)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML</a></li>
+    <li><a href="#">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- User authentication
+- Users can create and view transaction categories
+- Users can create and view transactions
+- Users can monitor total spend on each category
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]() IF YOU ARE REVIEWING THIS I CANT DEPLOY MY WORK CAUSE I DONT KNOW HOW TO COULD YOU PLEASE HELP ME WITH INSTRUCTONS OR PLAYBY PLAY? THANKS!!- [Video Documentation](https://www.loom.com/share/727e92f38770461c9faaaecf61df961a?sid=95f9b826-8cdf-4e1b-92fb-fe023f13860f)
```

Comment:
-  [ ] I tried to reach you on slack to share some links on how to deploy on render. I will share them below, please deploy your app as per the requirement.
https://docs.render.com/deploy-rails#deploy-manually
https://medium.com/@laura_brooks/how-to-deploy-a-ruby-on-rails-app-with-render-com-9a824193e636

Reply:
im still struggling with this